examples/ui.apps/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-themes/core-components-clean/styles/components/accordion/base.less

```diff
@@ -40,7 +40,6 @@
         color: inherit;
         font: inherit;
         cursor: pointer;
-        outline: inherit;
```

Comment:
Should `outline: inherited` be deleted from here too?
https://github.com/adobe/aem-core-wcm-components/blob/c6756447df4ebcfd66d133385502cae3135f87be/examples/ui.apps/src/content/jcr_root/apps/core-components-examples/components/accordion/clientlibs/amp/css/accordion.less#L57

Reply:
AMP mode is technically available for screen readers. That is probably very small percentage of use case however we should cover the same accessibility fix for both AMP and standard pages.